Tests/Tests/STPPaymentIntentTest.m

```diff
@@ -32,20 +32,27 @@ - (void)testIdentifierFromSecret {
 }
 
 - (void)testStatusFromString {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
     XCTAssertEqual([STPPaymentIntent statusFromString:@"requires_source"],
-                   STPPaymentIntentStatusRequiresSource);
+                    STPPaymentIntentStatusRequiresSource);
     XCTAssertEqual([STPPaymentIntent statusFromString:@"REQUIRES_SOURCE"],
-                   STPPaymentIntentStatusRequiresSource);
+                    STPPaymentIntentStatusRequiresSource);
```

Comment:
These can be updated to `.requiresPaymentMethod` and the `#pragmas` removed

Reply:
I was too hasty changing this test - we don't parse "requires_source" anymore, so I'm just going to assert `STPPaymentIntentStatusRequiresSource` == `STPPaymentIntentStatusRequiresAction`.  

We *do* still parse "requires_source_action", so I'm going to add assertions that it produces `STPPaymentIntentStatusRequiresAction`, as well as another `STPPaymentIntentStatusRequiresAction` == `STPPaymentIntentStatusRequiresSourceAction`